packages/collector/test/tracing/logger/instana-logger/worker.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * (c) Copyright IBM Corp. 2025
+ */
+
+/* eslint-disable no-console */
+
+'use strict';
+
+const { workerData } = require('worker_threads');
+
+console.log('Worker thread: Starting', workerData);
+
+const instana = require('../../../..')({
+  agentPort: workerData.agentPort
+});
```

Comment:
I have looked at the existing worker thread tests:
https://github.com/instana/nodejs/blob/v4.6.1/packages/collector/test/metrics/appWithWorkerThread/app.js#L28

https://github.com/instana/nodejs/blob/v4.6.1/packages/collector/test/metrics/appWithWorkerThread/node_modules/module-that-creates-a-worker-thread/index.js#L8

We require the custom NPM module. And on top we require Instana. A require call **does not** open a worker thread! I will fix this after we merged this one.